src/otx/algorithms/classification/configs/base/data/data_pipeline.py

```diff
@@ -20,7 +20,13 @@
 __resize_target_size = 224
 
 __train_pipeline = [
-    dict(type="LoadImageFromOTXDataset"),
+    dict(
+        type="LoadResizeDataFromOTXDataset",
+        load_img_cfg=dict(type="LoadImageFromOTXDataset", enable_memcache=False),  # To be cached after resize
+        resize_cfg=dict(type="Resize", size=__resize_target_size, downscale_only=True),
```

Comment:
Ah, I just realized that `ResizeTo` would be better here, too. Thanks :)

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/4629606/5758050b-142e-4e8d-b73a-dc1809abb39c)
This condition is already doing it so that we don't adopt another if-else here.